### ISSUE TITLE
Proposed Guidelines and Policies for Application pathways for quality enforcement

### ISSUE THREAD

**Issue by raghavrmadya** [filecoin-project/notary-governance#925]
Stakeholder | Public Open Datasets | Commercial Open Datasets | Private Datasets
| ---------- | ---------------------  | ---------------------------  | ---------------- |
| Data Owner | Verify the source of Open Dataset + [New registration form](https://github.com/filecoin-project/notary-governance/issues/922) (pending approval)  | Prove ownership, data size, and legitimacy + [New registration form](https://github.com/filecoin-project/notary-governance/issues/922) (pending approval) | Prove ownership through mandatory KYC, data size, and legitimacy + E-Fil+ Registration form |
| Data Preparer (DP) | KYC + Details on experience as a DP | KYC + Details on experience as a DP | Experience as a DP |
| Notary | Verify above steps + CID/retrieval report check + evidence of actioning on due diligence plan based on application | Verify above steps + CID/retrieval report check + evidence of actioning on due diligence plan based on application | Verify above steps + CID/retrieval report check + evidence of actioning on due diligence plan based on application

**Comment by raghavrmadya** [comment]
Based on multiple calls from the community to outline what T&T WG can do to enforce quality, the above table outlines what are the minimum requirements for each stakeholder. 

I would like to invite community input on this proposal.

**Comment by herrehesse** [comment]
Thank you, @raghavrmadya, for sharing this valuable information. I must commend the impressive improvements made to the new registration form for Public and Open datasets, thank you for this. I have a few questions regarding the various forms of stakeholders:

- I suggest that FIL+ be exclusively designated for "Public Open Datasets," while all commercial/enterprise sets should be moved to the FIL-E program.
- Datasets falling under FIL-E should undergo thorough examination to ensure their suitability for enterprise businesses. 
These entities can initially utilize storage capacity on the Filecoin network for a reduced cost or even for free, thanks to the "datacap grant." **However, it is expected that they will transition to paid agreements and start paying for the storage services provided by the storage providers, just like other companies, once the initial datacap grant period is completed.**


- It is crucial to limit the concept of free storage as it hinders progress into phase 3 of the Filecoin network.
- Open Public Datasets should collaborate closely with the FIL+ team to ensure optimal retrievability and indexing, making them truly valuable for the ecosystem. They should strive to be a clear added value, avoiding merged sets, excessive copies, and similar practices, and work towards integration with https://openpanda.io/.

To promote the growth and effectiveness of the Filecoin ecosystem, we need to swiftly prevent random entities from acquiring valuable datacap for irrelevant data solely to increase personal profit through the datacap multiplier.

In summary:

- OR add value to the network by preparing and distributing datasets from the wide range of nearly 200 scientific datasets available on [OpenPanda](https://openpanda.io/), while ensuring near-perfect retrieval and indexing capabilities.
- OR showcase the network's capabilities to onboard enterprise clients and transition them into paid storage, thereby contributing value to the ecosystem.
- BUT reject random data, such as YouTube videos, CCTV footage, or similar content, that is uploaded solely to acquire datacap. Entities seeking to store data on Filecoin should connect with storage providers (SPs) and pay for their services.

Let's prioritise these actions to enhance the Filecoin ecosystem and drive its progress.

**Comment by kernelogic** [comment]
Does Quality Phase apply to FIL-E too or no? We have seen a lot dispute on FIL+ LDNs but not many (or at all) on FIL-E LDNs. My concern is **IF** FIL-E is more relaxed people would flock to it to escape quality enforcement.

**Comment by herrehesse** [comment]
@kernelogic Fair thought, as far as I am aware @kevzak is holding an even higher standard to FIL-E participants. As it should.

**Comment by kernelogic** [comment]
That's good to know. Thanks.

**Comment by liyunzhi-666** [comment]
> Does Quality Phase apply to FIL-E too or no? We have seen a lot dispute on FIL+ LDNs but not many (or at all) on FIL-E LDNs. My concern is **IF** FIL-E is more relaxed people would flock to it to escape quality enforcement.

The KYB/KYC process and content of the current E-FIL+ is of a higher standard than FIL+.

**Comment by Chuangshi1** [comment]
Will it start with #922 ?

**Comment by Chris00618** [comment]
>   These entities can initially utilize storage capacity on the Filecoin network for a reduced cost or even for free, thanks to the "datacap grant." **However, it is expected that they will transition to paid agreements and start paying for the storage services provided by the storage providers, just like other companies, once the initial datacap grant period is completed.**
> * It is crucial to limit the concept of free storage as it hinders progress into phase 3 of the Filecoin network.

In support of this, it is proposed that each enterprise should be allowed to apply for a maximum quota of 3P; at the same time, a quota cap of 5P open public datasets should be set for each DP or applicant. For the excess, a market pricing mechanism should be formed, with SPs and demand-side negotiating the pricing.

**Comment by MRJAVAZHAO** [comment]
I think the community should have some detailed requirements on some discussions.

Public Open Datasets

- Can public data sets be stored repeatedly by different clients?
- The maximum number of backups that can be stored?

Commercial Open Datasets/Private Datasets

- How to prove the total amount of data?

**Comment by spaceT9** [comment]
It  would be great to define the dataset types clearly

**Comment by carol19920101** [comment]
saw a lot of discussions and optimizaiton proposal in community recently. 
Just wonder if we can integrate it into one?
or I can take #925 as the thread of new principles followup?

**Comment by HiFil** [comment]
Too many abuse issues right now. Suggest a one-month moratorium on approving new applications until the rules are agreed upon.

If agreement cannot be reached, Highly recommended that the maximum amount of datacap for one application should be reduced.

**Comment by Chris00618** [comment]
It's too early for FIL+ to enter the data quality phase, getting sufficiently decentralized data orders into the network through simpler rules is the way to go.
I think @raghavrmadya 's proposal will make the current situation even worse.  FIL+ needs a major surgery, not optimization. Please refer to our proposal as below:

https://github.com/filecoin-project/notary-governance/issues/928
